packages/ckeditor5-alignment/src/utils.js

```diff
@@ -44,3 +51,63 @@ export function isDefault( alignment, locale ) {
 		return alignment === 'left';
 	}
 }
+
+/**
+ * Brings the configuration to the common form, an array of objects.
+ *
+ * @param {Array.<String|Object>} configuredOptions Alignment plugin configuration.
+ * @returns {Array.<Object>} Normalized object holding the configuration.
+ */
+export function normalizeAlignmentOptions( configuredOptions = [] ) {
+	return configuredOptions.map( normalizeOption );
+}
+
+// @private
+function normalizeOption( option, index, allOptions ) {
+	let optionObj;
+
+	if ( typeof option == 'string' ) {
+		optionObj = Object.assign( {}, { name: option } );
+	} else {
+		optionObj = Object.assign( {}, defaultOptions[ index ], option );
+	}
+
+	const succeedingOptions = allOptions.slice( index + 1 );
+	const nameAlreadyExists = succeedingOptions.some(
+		item => {
+			const optionName = item.name || item;
+
+			return optionName == optionObj.name;
+		} );
```

Comment:
```suggestion
	const nameAlreadyExists = succeedingOptions.some( item => {
		const optionName = item.name || item;

		return optionName == optionObj.name;
	} );
```

Maybe we could do normalization from \`"string" to { name: "string" }\` in the first loop and then in the next loop normalize it?

Reply:
Done.